contentcuration/contentcuration/models.py

```diff
@@ -476,6 +487,8 @@ def get_channel(self):
             return None
 
     def save(self, *args, **kwargs):
+        self.changed = self.changed or len(self.get_changed_fields()) > 0
```

Comment:
Why not just base our `changed` state solely on whether any fields have been modified?